src/view/frontend/web/css/tailwind/package.json

```diff
@@ -4,22 +4,22 @@
   "description": "Tailwind CSS compiler",
   "main": "tailwind.config.js",
   "dependencies": {
-    "@tailwindcss/forms": "^0.2.1",
-    "autoprefixer": "^10.2.1",
+    "@tailwindcss/forms": "^0.3.3",
+    "autoprefixer": "^10.3.1",
     "csso-cli": "^3.0.0",
-    "postcss-import": "^12.0.1",
-    "postcss-nested": "^4.2.3",
-    "tailwindcss": "^2.0.2",
-    "terser": "^4.7.0"
+    "postcss-import": "^14.0.2",
+    "postcss-nested": "^5.0.6",
+    "tailwindcss": "^2.2.7",
+    "terser": "^5.7.1"
   },
   "devDependencies": {
-    "postcss": "^8.2.4",
+    "postcss": "^8.3.6",
     "postcss-cli": "^8.3.1"
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "npx postcss tailwind-source.css -o tailwind-output.css",
-    "minify": "npx csso tailwind-output.css --output ../styles.css",
+    "build": "node_modules/postcss-cli/bin/postcss tailwind-source.css -o tailwind-output.css",
```

Comment:
Having csso-cli and postcss/postcss-cli in the "dependencies" would automatically make it use these packages, wouldn't it?
Or would it first try to use a globally installed version that might be too old?

If this works, I don't mind of course, just trying to understand the reasoning :)

Reply:
@wigman No, the local package won't be used when we use it on commands.  Commands are always in the global scope. If the global package is not present, then it will complain the command not found. So in order to force the local package use, we should use the above trick.